<launch>
  <node pkg="move_base" type="move_base" name="move_base" output="screen">
    <rosparam file="$(find navigation)/config/global_costmap.yaml" command="load" ns="global_costmap" />
    <rosparam file="$(find navigation)/config/local_costmap.yaml" command="load" ns="local_costmap" />
    <param name="base_global_planner" value="navfn/NavfnROS" />
    <param name="base_local_planner" value="base_local_planner/TrajectoryPlannerROS" />
  </node>
</launch>
